src/components/MoneyRequestHeader.tsx

```diff
@@ -60,11 +64,26 @@ type MoneyRequestHeaderProps = MoneyRequestHeaderOnyxProps & {
     onBackButtonPress: () => void;
 };
 
+type NoPendingStatusBarProps = {isPending: false};
+
+type PendingStatusBarProps = {
+    isPending: true;
```

Comment:
Don't we return `false` for this in the default case for `getStatusBarProps`?

Reply:
yes, we do, is there any problem with that?

Reply:
Sorry, I trust you on typescript more, but my understanding was that this says `isPending` is always `true`?

Reply:
Oh I see, it's the `NoPendingStatusBarProps` above. Nevermind me.